rfcs/text/0031-risk-fields.md

```diff
@@ -31,13 +43,74 @@ Stage 2: Add or update all remaining field definitions. The list should now be e
 <!--
 Stage 1: Describe at a high-level how these field changes will be used in practice. Real world examples are encouraged. The goal here is to understand how people would leverage these fields to gain insights or solve problems. ~1-3 paragraphs.
 -->
+As mentioned previously, we have released an experimental feature called Host Risk Score in the Security solution recently. As of 7.16, the feature has some real estate on the Overview page and the Alert Flyout within the Security solution, as documented [here](https://www.elastic.co/guide/en/security/8.0/host-risk-score.html). In 8.1, users will also be able to see host risk information on the Hosts page and Host Details page. 
+
+In addition to Host Risk Score, there is work currently in progress to introduce a Users page in the Security solution and a User Risk Scoring capability. Entities at risk is a new concept for users of the Security solution. Defining and normalizing this concept of entity risk using the `risk` fields will be crucial for users to get the most out of the Host and User Risk Scoring capabilities when they go GA.
+
+Furthermore, these `risk` fields will provide users with an additional vector to filter, sort and correlate information within the Security solution. For example, users will be able to start investigations by running queries like the following:
+* "Show me the most critical and high-risk Windows hosts in my environment"
+* "Show me the activity that contributed towards making Host X high-risk"
+* "Show me how the risk of Host X changed over time"
+* "Show me Critical and high-risk users on Host X"
+
 
 ## Source data
 
 <!--
 Stage 1: Provide a high-level description of example sources of data. This does not yet need to be a concrete example of a source document, but instead can simply describe a potential source (e.g. nginx access log). This will ultimately be fleshed out to include literal source examples in a future stage. The goal here is to identify practical sources for these fields in the real world. ~1-3 sentences or unordered list.
 -->
-
+The Host and User Risk Score views in the Security solution are/will be backed by indices produced as a result of running [transforms](https://www.elastic.co/guide/en/elasticsearch/reference/current/put-transform.html) on a variety of data sources, not limited to the alerts data streams. An example document produced by the Host Risk Score transform in the absence of ECS `risk` fields looks as follows:
```

Comment:
Instead of nesting, could `risk.*` be a top-level field and have a field that specifies the `risk.type` as `host`, `user`, etc?

Unless both `host.risk.*` and `user.risk.*` appear together in an event, avoiding nesting might simplify the schema and queries (e.g., I only query `risk.score` instead of `host.risk.score` AND `user.risk.score`).

Reply:
@ebeahan Interesting callout about `risk_score` not accepting a float value in other places. Totally agree about getting consensus on capturing `risk` in the Security solution. I'll create a Discuss issue for this.

re: nesting, it is possible that `host.risk.*` and `user.risk.*` might appear in the same event. And the risk score for the two will likely be different.